src/configure/configure.ts

```diff
@@ -484,34 +498,22 @@ class Orchestrator {
         telemetryHelper.setTelemetry(TelemetryKeys.SubscriptionId, this.inputs.subscriptionId);
     }
 
-    private async getSelectedPipeline(repoAnalysisResult: RepositoryAnalysis): Promise<void> {
-        extensionVariables.templateServiceEnabled = true;
+    private async getTemplatesByRepoAnalysis(repoAnalysisResult: RepositoryAnalysis): Promise<void> {
+
         let appropriatePipelines: PipelineTemplate[] = [];
+        const remotePipelines: PipelineTemplate[] = await vscode.window.withProgress(
```

Comment:
I guess you have removed the check for `templateServiceEnabled`. But if you want to do so then you should do across the complete code and remove the variable definition as well. Otherwise it is not complete cleanup

Reply:
Did you handle this comment?

Reply:
yes, it was there only in model.ts. Have removed it.